protostar/cheatable_starknet/cheatable_exception.py

```diff
@@ -0,0 +1,7 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class CheatableException(Exception):
```

Comment:
Why is it "cheatable"?

Reply:
`CheaterException` makes more sense?

Reply:
TransactionException? I think it better captures the class of errors we want to capture

Reply:
TransactionRevertException maybe?

Reply:
[rename exception again](https://github.com/software-mansion/protostar/pull/1404/commits/7b51366559cd7ff6aee5d2df347dd068a68973ea)